ansible/roles/caddy/templates/caddyfile.j2

```diff
@@ -0,0 +1,19 @@
+{
+  skip_install_trust
+
+  log {
+    format filter {
+      wrap json
+      fields {
+        common_log delete
+        request>headers>Authorization delete
+      }
+    }
+  }
+
+{% if caddy_email is defined %}
+  email {{ caddy_email }}
```

Comment:
This is for ACME. If something happens to our TLS, that email can prove that we own these keys.